test/AppNotificationTests/AppNotificationHelpers.h

```diff
@@ -0,0 +1,237 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#include "pch.h"
+
+namespace winrt
+{
+    using namespace winrt::Windows::ApplicationModel::Activation;
+    using namespace winrt::Windows::Foundation;
+    using namespace winrt::Microsoft::Windows::AppNotifications;
+    using namespace winrt::Windows::Foundation;
+}
+
+namespace AppNotificationHelpers {
```

Comment:
Helpers? 🤨

Reminder: That's one of my trigger words...

This is just code for use by AppNotification tests? Then why not `namespace AppNotification::Test`?